nvflare/fuel/hci/server/hci.py

```diff
@@ -32,8 +32,19 @@ class _MsgHandler(socketserver.BaseRequestHandler):
     ServerCommandRegister.
     """
 
+    logger = logging.getLogger("_MsgHandler")
```

Comment:
These lines should be placed into the constructor, instead of class vars.

Reply:
The counter needs to be class vars. The logger is moved to the contructor.